src/components/Menu.js

```diff
@@ -1,8 +1,128 @@
+  }
+
+  handleClick(idx) {
+    this.setState({
+      ...this.state,
+      menuOnDisplay: this.state.menulist[idx].name,
+      menuItems: this.state.menulist[idx].items,
+      idx: idx
+    })
+  }
+  onSortUp() {
+    let sorted = [...this.state.menuItems].sort((a, b) => {
+      return a.price - b.price;
+    });
+    this.setState({
+      menuItems: sorted
+    });
+  }
+  onSortDown() {
+    let sorted = [...this.state.menuItems].sort((a, b) => {
+      return b.price - a.price;
+    });
+    this.setState({
+      menuItems: sorted
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="header">
+          <Image
+            src='/pinks.jpg'
+            style={{
+              display: "block",
+              margin: "auto",
+              width: "158px",
+              height: "148px",
+            }}
+            rounded
+          />
+          <Dropdown as={ButtonGroup}  style={{"margin-right": "10px"}}>
+            <Button variant="success">All Menus</Button>
+            <Dropdown.Toggle split variant="success" id="dropdown-split-basic" />
+            <Dropdown.Menu>
+              {
+                this.state.menulist.map((menu, idx) => (
+                  <Dropdown.Item onClick={() => this.handleClick(idx)} id={idx} >{menu.name.toUpperCase()}</Dropdown.Item>
+                ))
+              }
+            </Dropdown.Menu>
+          </Dropdown>
+          <div style={{"font-weight": "bold"}}>Sort by price:</div>
+          <Button style={{"margin-right": "10px"}} onClick={this.onSortUp.bind(this)}>ascending <FontAwesomeIcon icon={faSortUp} /></Button>
+          <Button onClick={this.onSortDown.bind(this)}>descending <FontAwesomeIcon icon={faSortDown} /></Button>
+        </div>
+        
+        <div className='category'>
+        {
+          this.state.menuItems
+          .map(({ name, imageUrl, price, description, id }) => (
+              <div key={id}>
+                <Card style={{ padding: "30px" }}>
+                  <Row>
+                    <Col xs={3} md={2}>
+                      <Card.Header>
+                        <Image
+                          src={imageUrl}
+                          style={{
+                            display: "block",
+                            margin: "auto",
+                            width: "158px",
+                            height: "148px",
+                          }}
+                          rounded
+                        />
+                      </Card.Header>
+                    </Col>
+                    <Col xs={12} md={8}>
+                      <Card.Title style={{ "font-size": "30px" }}>{name.charAt(0).toUpperCase() + name.slice(1)}</Card.Title>
+                      <Card.Title style={{ "font-size": "30px" }}>
+                        Price: <Badge variant="info">$ {price}</Badge>{" "}
+                      </Card.Title>
+                      <Card.Text style={{ "font-size": "25px" }}>{description}</Card.Text>
```

Comment:
I think you need to reformat the layout of the card because the description text is clipping over the image on my machine.

Reply:
The text is still clipping.